src/main/java/org/hobbit/core/components/utils/SystemResourceUsageRequester.java

```diff
@@ -65,7 +64,9 @@ public static SystemResourceUsageRequester create(PlatformConnector connector, S
     protected Gson gson = new Gson();
 
     protected SystemResourceUsageRequester(Channel cmdChannel, Channel incomingChannel, String responseQueueName,
-            QueueingConsumer responseConsumer, String sessionId) {
+            String sessionId) throws IOException {
```

Comment:
It is not very nice to have a constructor that throws an exception. Could you please move the two lines that could cause this exception into the `create` method as it has been implemented, before?